<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <widget class="QLabel" name="VideoHolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1831</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AlertTitle">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>1831</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>&quot;Arial&quot;</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
border-style:outset;
border-radius:10px;
color: rgb(250, 255, 255);
font: 14pt \&quot;Arial\&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>        ALERT</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="AlertText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>1001</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>&quot;Arial&quot;</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
color: rgb(0, 0, 0);
font: 14pt \&quot;Arial\&quot;;
</string>
    </property>
    <property name="text">
     <string> alert text</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="VerifiedBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(73, 199, 41);
border-style:outset;
border-radius:10px;
color: rgb(250, 255, 255);
font: 14pt \&quot;Arial\&quot;;
</string>
    </property>
    <property name="text">
     <string>Verified</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FalseBtn">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(250, 255, 255);
background-color: rgb(199, 73, 41);
border-style:outset;
border-radius:10px;
font: 14pt \\&quot;Arial\\&quot;;</string>
    </property>
    <property name="text">
     <string>False</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VerifiedBtn_2">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(73, 41, 199);
border-style:outset;
border-radius:10px;
color: rgb(250, 255, 255);
font: 14pt \&quot;Arial\&quot;;
</string>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
